charts/argocd-apps/Chart.yaml

```diff
@@ -2,4 +2,4 @@
 apiVersion: v2
 name: argocd-apps
 description: Installs ArgoCD applications into the cluster-services namespace
-version: 0.3.10
+version: 0.3.11
```

Comment:
No need to bump this version but don't worry about it :)

Reply:
Oh wait actually it's kinda the other way around — feel free to merge this and try it out though!

Reply:
🤞🏻